Portfolio/index.css

```diff
@@ -1,4 +1,7 @@
 console.log('Hello World');

```

Comment:
Ensure there is a proper opening bracket '{' before defining styles for the 'h1' tag.